.coveragerc

```diff
@@ -11,3 +11,8 @@ omit =
     # TODO(#13935): Remove python_utils from the list once the codebase is
     # run under Python 3.
     core/python_utils.py
+
+[report]
+exclude_lines =
+    pragma: no cover
```

Comment:
Do you need to add this? I thought this was excluded by default.

Reply:
![Screenshot 2022-01-17 102809](https://user-images.githubusercontent.com/12553576/149752904-a209425e-29de-4761-993b-2b50898c71ae.jpg)
https://coverage.readthedocs.io/en/coverage-4.3.3/excluding.html#advanced-exclusion
as explained here!!

Reply:
0kay, thanks!

Reply:
One more thing, can you add an explanatory comment, explaining why we add `@overload` and also why we add `pragma: no cover`

Reply:
Done!